src/test/java/htsjdk/samtools/reference/IndexedFastaSequenceFileTest.java

```diff
@@ -56,7 +57,8 @@
                 { new IndexedFastaSequenceFile(SEQUENCE_FILE) },
                 { new IndexedFastaSequenceFile(SEQUENCE_FILE_NODICT) },
                 { new IndexedFastaSequenceFile(SEQUENCE_FILE.toPath()) },
-                { new IndexedFastaSequenceFile(SEQUENCE_FILE_NODICT.toPath()) }};
+                { new IndexedFastaSequenceFile(SEQUENCE_FILE_NODICT.toPath()) },
+                { new BlockCompressedIndexedFastaSequenceFile(SEQUENCE_FILE_GZ.toPath())}};
```

Comment:
can you rename this variable to make it clear that it's really a block zipped file and not just a gz file.

Reply:
Renamed to `SEQUENCE_FILE_BGZ` to clarify